src/options.c

```diff
@@ -380,8 +380,9 @@ void optionsParse(int argc, char *argv[])
             opt.display = optarg;
             break;
         case 'd':
-            opt.delay_selection = *optarg == 'b';
-            if (opt.delay_selection)
+
```

Comment:
Most other case statements don't have an immediate blank line. Was this intentional?